apps/server/src/shared/domain/entity/system.entity.ts

```diff
@@ -100,7 +171,7 @@ export class System extends BaseEntityWithTimestamps {
 	config?: Record<string, unknown>;
 
 	@Property({ nullable: true })
-	ldapConfig?: Record<string, unknown>;
+	ldapConfig?: LdapConfig;
```

Comment:
Do this work for MircoORM if a class is added with constructor?

Reply:
I know that the constructor can be free used. But my question is, can the ORM load it from DB. It is only a "normal class" for the ORM. Yes it has some decorators but the class is not decorated. I am not sure if we have the case before. But we have some trouble with sub documents in the past that can not handle from the ORM in the right way.